File: app/src/main/java/cc/smartcash/wallet/Models/ExplorerApiTransactionOutput.java
package cc.smartcash.wallet.Models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

import java.io.Serializable;

@JsonIgnoreProperties(ignoreUnknown = true)
public class ExplorerApiTransactionOutput implements Serializable {

    @JsonProperty("Txid")
    private String txid;
    @JsonProperty("Index")
    private int index;
    @JsonProperty("Address")
    private String address;
    @JsonProperty("Value")
    private double value;
    @JsonProperty("Json")
    private String json;

    public String getTxid() {
        return txid;
    }

    public void setTxid(String txid) {
        this.txid = txid;
    }

    public int getIndex() {
        return index;
    }

    public void setIndex(int index) {
        this.index = index;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public double getValue() {
        return value;
    }

    public void setValue(double value) {
        this.value = value;
    }

    public String getJson() {
        return json;
    }

    public void setJson(String json) {
        this.json = json;
    }

}

File: app/src/main/java/cc/smartcash/wallet/Models/ExplorerApiTransactionOutput.kt
package cc.smartcash.wallet.Models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

import java.io.Serializable

@JsonIgnoreProperties(ignoreUnknown = true)
class ExplorerApiTransactionOutput : Serializable {

    @JsonProperty("Txid")
    var txid: String? = null

    @JsonProperty("Index")
    var index: Int = 0

    @JsonProperty("Address")
    var address: String? = null

    @JsonProperty("Value")
    var value: Double = 0.toDouble()

    @JsonProperty("Json")
    var json: String? = null

}
